demo/index.html

```diff
@@ -159,6 +166,69 @@
           .catch(onError);
         });
 
+        startNotificationsButton.addEventListener('click', function() {
+          heartRateDevice.request().then(function(device) {
+            return heartRateMeasurement.startNotifications().then(function() {
+              template.text = device.name + ' Notifications session has started.';
+              progressBar.indeterminate = false;
+              heartRateMeasurement.addEventListener('value-changed', parseHeartRate);
```

Comment:
This could go through a [Polymer change observer](https://www.polymer-project.org/1.0/docs/devguide/properties.html#change-callbacks) instead of a JS event